MatrixSDK/JSONModels/AutoDiscovery/MXWellKnown.m

```diff
@@ -16,6 +16,13 @@
 
 #import "MXWellKnown.h"
 
+static NSString *const kMXHomeServerKey = @"m.homeserver";
```

Comment:
Technically I believe homeserver is considered a single word so in theory it would probably be `kMXHomeserverKey`. But as there are `homeServer` references elsewhere it's not worth changing.

Reply:
Yeah, I noticed that in the docs but didn't like how it looked next to the others 🙂